platzky/admin/fake_login.py

```
@@ -0,0 +1,53 @@
+"""
+Fake login functionality for development environments only.
+
+WARNING: This module provides fake login functionality and should NEVER be used in production
+environments as it bypasses proper authentication and authorization controls.
+"""
+from typing import Any
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)